extensions/warp-ipfs/src/store/message/task.rs

```diff
@@ -3482,6 +3516,14 @@ async fn message_event(
                     }
                 }
                 ConversationUpdateKind::AddedIcon | ConversationUpdateKind::RemovedIcon => {
+                    if !this.document.creator.as_ref().is_some_and(|c| c == sender)
+                        && !this
+                            .document
+                            .permissions
+                            .has_permission(sender, GroupPermission::EditGroupInfo)
+                    {
+                        return Err(Error::Unauthorized);
+                    }
```

Comment:
A check should be added to determine if the conversation is a group before checking the permission